starter/script.js

```diff
@@ -1 +1,48 @@
-// put code here!
+
+x = 0;
+while(x<3){
+var name = prompt("Hello, What is your name?");
```

Comment:
Watch your indentation! `var name` is *inside* `while(x<3){`. So indent it to show visually that it's inside:

```
while(x<3){
  var name = ...
```